--- bes-3.17.0/configure.ac.orig	2016-02-03 19:37:02.308179475 +0100
+++ bes-3.17.0/configure.ac	2016-03-26 09:16:06.978452939 +0100
@@ -406,7 +406,6 @@
 	[with_openjpeg_prefix=$ac_bes_dependencies_prefix])
 
 save_LIBS=$LIBS
-LIBS="-L$with_openjpeg_prefix/lib $LIBS"
 AC_CHECK_LIB(openjp2, opj_version, [OPENJPEG_FOUND="yes"], [OPENJPEG_FOUND="no"], [-lm])
 LIBS=$save_LIBS
 
@@ -457,12 +456,12 @@
 
 
 ac_cfits_save_LDFLAGS="$LDFLAGS"
-LDFLAGS="$LDFLAGS -L$CFITS_PATH_LIBDIR"
+AS_IF([test -n "$CFITS_PATH_LIBDIR"],[LDFLAGS="$LDFLAGS -L$CFITS_PATH_LIBDIR"])
 AC_CHECK_LIB(cfitsio, fits_is_url_absolute, 
 
 [ac_cfits_ok='yes'
 CFITS_LIBS="-lcfitsio"
-CFITS_LDFLAGS="-L${CFITS_PATH_LIBDIR}" 
+AS_IF([test -n "$CFITS_PATH_LIBDIR"], [CFITS_LDFLAGS="-L${CFITS_PATH_LIBDIR}"], [CFITS_LDFLAGS=""])
 AC_SUBST([CFITS_LDFLAGS])
 AC_SUBST([CFITS_LIBS])],
 
@@ -474,7 +473,7 @@
 CFITS_CPPFLAGS=
 ac_cfits_h='no'
 ac_cfits_save_CPPFLAGS="$CPPFLAGS"
-CPPFLAGS="$CPPFLAGS -I$CFITS_PATH_INC"
+AS_IF([test -n "$CFITS_PATH_INC"], [CPPFLAGS="$CPPFLAGS -I$CFITS_PATH_INC"])
 AC_CHECK_HEADERS([fitsio.h],
   [ac_cfits_h='yes'],
   [ac_cfits_h='no']
@@ -482,7 +481,7 @@
 CPPFLAGS=$ac_cfits_save_CPPFLAGS
 
 AS_IF([test "$ac_cfits_h" = 'yes' ],
-  [CFITS_CPPFLAGS="-I$CFITS_PATH_INC"]
+  [AS_IF([test -n "$CFITS_PATH_INC"], [CFITS_CPPFLAGS="-I$CFITS_PATH_INC"], [CFITS_CPPFLAGS=""])]
 )
 AC_SUBST([CFITS_CPPFLAGS])
 
@@ -518,8 +518,8 @@
             [HDFEOS2_DIR="$ac_bes_dependencies_prefix"])
 
 AS_IF([test -n "$HDFEOS2_DIR"],
-    [HDFEOS2_CPPFLAGS="-I$HDFEOS2_DIR/include -DUSE_HDFEOS2_LIB"
-    HDFEOS2_LDFLAGS="-L$HDFEOS2_DIR/lib"
+    [AS_IF([test x"$HDFEOS2_DIR" = xyes], [HDFEOS2_CPPFLAGS="-DUSE_HDFEOS2_LIB"], [HDFEOS2_CPPFLAGS="-I$HDFEOS2_DIR/include -DUSE_HDFEOS2_LIB"])
+    AS_IF([test x"$HDFEOS2_DIR" = xyes], [HDFEOS2_LDFLAGS=], [HDFEOS2_LDFLAGS="-L$HDFEOS2_DIR/lib"])
     HDFEOS2_LIBS="-lhdfeos -lGctp"
     AC_MSG_NOTICE([HDFEOS2 set to true])
     AM_CONDITIONAL([HDFEOS2], [true])],
--- bes-3.17.0/conf/hdf4.m4.orig	2015-04-05 05:21:13.650708879 +0200
+++ bes-3.17.0/conf/hdf4.m4	2016-03-28 20:38:01.114526832 +0200
@@ -63,7 +63,7 @@
     ],
     [
       AC_MSG_NOTICE(HDF4_PATH_LIBDIR is NOT set)
-      for ac_hdf4_libdir in /usr/local/hdf4.2r1/lib64 /opt/hdf4.2r1/lib64 \ 
+      for ac_hdf4_libdir in "" /usr/local/hdf4.2r1/lib64 /opt/hdf4.2r1/lib64 \ 
        /usr/hdf4.2r1/lib64 /usr/local/lib64/hdf4.2r1 /opt/lib64/hdf4.2r1 \
        /usr/lib64/hdf4.2r1 /usr/local/hdf/lib64/ /opt/hdf/lib64 \
        /usr/hdf/lib64 /usr/local/lib64/hdf /opt/lib64/hdf /usr/lib64/hdf \
@@ -101,7 +101,7 @@
     ],
     [
        AC_MSG_NOTICE(HDF4_PATH_INC is NOT set)
-      for ac_hdf4_incdir in /usr/local/hdf4.2r1/include /opt/hdf4.2r1/include \
+      for ac_hdf4_incdir in "" /usr/local/hdf4.2r1/include /opt/hdf4.2r1/include \
        /usr/hdf4.2r1/include /usr/local/include/hdf4.2r1 \
        /opt/include/hdf4.2r1 /usr/include/hdf4.2r1 /usr/local/hdf/include \
        /opt/hdf/include /usr/hdf/include /usr/local/include/hdf \
